CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master
 
+## 0.7.0
```

Comment:
Can you remove this from the PR? We'll update this when we are ready to publish a new version.